nextflow/MaveDB/nf_modules/licences.nf

```diff
@@ -0,0 +1,74 @@
+process get_licence {
+  // Check if files are open access (error out if not)
+
+  errorStrategy 'ignore'
+
+  input:  path mappings
+  output: tuple path(mappings), path('LICENCE.txt')
+
+  """
+  #!/usr/bin/env python3
+  import urllib.request
+  import json
+
+  urn = "urn:mavedb:${mappings.simpleName}"
+  url = f"https://api.mavedb.org/api/v1/scoresets/{urn}"
+  res = urllib.request.urlopen(url).read()
+  res = json.loads(res)
+
+  f = open("LICENCE.txt", "w")
+  f.write(res['license']['shortName'])
+  f.close()
+  """
+}
+
+workflow get_files_by_licence {
```

Comment:
should we have workflow in a module file? should it be best to have 2 modules instead of calling a module from this workflow?